examples/icd_code_prediction/README.md

```diff
@@ -0,0 +1,46 @@
+## A ICD Code Prediction Demo
+
+This demo shows how we can build pipeline to predict ICD-10 codes for medical notes.
+
+The [International Classification of Diseases (ICD)](https://www.cdc.gov/nchs/icd/index.htm) is the classification used to code and classify mortality data from death certificates. We treat the ICD code prediction as a sequence classification task. That is, given an medical note, the model output its ICD code.
+
+A key component of this demo is `ICDCodingProcessor`, which is implemented based on [AkshatSurolia's pre-trained model](https://huggingface.co/AkshatSurolia/ICD-10-Code-Prediction).
+
+## Install extra dependencies
+
+We need to install `forte.elastic` and `forte.elastic`.
```

Comment:
Why are we installing `forte.elastic` when it is not used in the example?

Reply:
I copied that from `examples/mimic_iii/medical_pipeline.py`.

When I uninstall that package, the code works fine. I will remove that dependency.